features/frameworks/utest/utest/utest_serial.h

```diff
@@ -19,17 +19,11 @@
  ****************************************************************************
  */
 
-#if DEVICE_SERIAL
-
 #ifndef UTEST_SERIAL_H
 #define UTEST_SERIAL_H
 
-#include "greentea-client/greentea_serial.h"
-
-#define utest_printf(...) greentea_serial->printf(__VA_ARGS__)
+extern "C" int utest_printf(const char *format, ...);
```

Comment:
Could you rename this file (and its new C++ file) to `utest_console` or `utest_print(f)` or something?

Reply:
Is it really necessary? I didn't want to have to rename it all files that use it. Unless you will be happy with `#define utest_printf(...) utest_print(__VA_ARGS__) `

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/72e390ca702f6528c2c52c576808d7a4decbbfd5..037924aeeb7c93705576fa38b71b19fa5db44bcd).